webapp/.eslintignore

```diff
@@ -0,0 +1 @@
+mattermost-webapp/*
```

Comment:
I see this is already ignored though. Nevermind about the `mattermost-webapp` linting comment then. But I still think we should keep all previously enabled eslint rules enabled, unless we really need to disable them